.github/workflows/prbuild.yaml

```diff
@@ -90,6 +90,66 @@ jobs:
           steps: ${{ toJson(steps) }}
           channel: '#contour-ci-notifications'
         if: ${{ failure() && github.ref == 'refs/heads/main' }}
+  e2e-contourconfiguration:
```

Comment:
You should be able to edit the matrix in the existing `e2e` job to have it run both with and without the CRD -- see e.g. https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-running-with-multiple-operating-systems (ignore the operating system bit, but it shows having two different dimensions for the matrix). Will avoid having to duplicate the entire job config. So something like:

```
matrix:
    kubernetes_version: ["kubernetes:latest", "kubernetes:n-1", "kubernetes:n-2"]
    use_config_crd: [true, false]
...
```